_notices/rgn0024.md

```diff
@@ -0,0 +1,48 @@
+---
+layout: notice
+parent: RAPIDS General Notices
+grand_parent: RAPIDS Notices
+nav_exclude: true
+notice_type: rgn
+# Update meta-data for notice
+notice_id: 24 # should match notice number
+notice_pin: false # set to true to pin to notice page
+title: "Updated WSL2 Installation Method"
+notice_author: RAPIDS Ops
+notice_status: Completed
+notice_status_color: green
+# 'notice_status' and 'notice_status_color' combinations:
+#   "Proposal" - "blue"
+#   "Completed" - "green"
+#   "Review" - "purple"
+#   "In Progress" - "yellow"
+#   "Closed" - "red"
+notice_topic: Breaking Change
+notice_rapids_version: "v22.02+"
+notice_created: 2022-02-17
+# 'notice_updated' should match 'notice_created' until an update is made
+notice_updated: 2022-02-17
+---
+
+## Overview
+
+The installation method for a RAPIDS compatible WSL2 instance has changed with the release of NVIDIA's WSL2 
+capable Production Drivers.  If you already have RAPIDS running on WSL2, Please 
+[update your drivers](https://www.nvidia.com/Download/index.aspx?lang=en-us) 
+accordingly for continued EA support with RAPIDS `v22.02`.  If you are installing RAPIDS inside WSL2 for the 
+first time, please follow the updated instructions below:
+	1. Please use this MS Blog for to install WSL2: https://docs.microsoft.com/en-us/windows/wsl/install
+	2. Install the latest production drivers, which are WSL2 capable: https://www.nvidia.com/Download/index.aspx?lang=en-us
+	3. Inside WSL2:
+		a. Install conda or docker environments
+        b. Install RAPIDS as normal with your environment options of choice.
+
+Please do not install an additional NVIDIA drivers inside WSL2.  It is no longer necessary.
+
+## Status
+
+- **12-Feb-2022** - EA support continues.
+
+## Impact
+
+Users intending to run RAPIDS `v22.02+`.  
```

Comment:
This looks like an incomplete sentence--we should fill out the rest of the impact, e.g. "users . . .  should use the latest instructions as defined above"

Reply:
addressed in commit d089b67